public/app/features/explore/QueryField.tsx

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 import React, { Context } from 'react';
 import ReactDOM from 'react-dom';
+import { List } from 'immutable';
```

Comment:
Is there a way to do this without anything from `immutable`. I know it's an explicit dependency now (because of slate), but it's not used anywhere else in the project. Not a blocker.